ci/test/00_setup_env_native_multiprocess.sh

```diff
@@ -10,5 +10,5 @@ export CONTAINER_NAME=ci_native_multiprocess
 export PACKAGES="cmake python3 llvm clang"
 export DEP_OPTS="DEBUG=1 MULTIPROCESS=1"
 export GOAL="install"
-export BITCOIN_CONFIG="--with-boost-process --enable-debug CC=clang CXX=clang++" # Use clang to avoid OOM
+export BITCOIN_CONFIG="--with-boost-process --enable-debug CC=clang-16 CXX=clang++-16" # Use clang to avoid OOM
```

Comment:
why clang-16 is enforced here?

Reply:
To reduce ambiguity. Currently, both Clang 16 and distro-supplied Clang are installed and you have to specify Clang 16 using the suffix `-16` while the variables above would be using distro-supplied clang.

This ambiguity has been sorted by removing distro-supplied Clang and LLVM but I'd prefer explicitly stating the compiler version we're expecting, esp. when we're outwardly deciding the version.